TrustWallet/wallet-core
# FIX install-dependencies with Swift 5 / Xcode 10.2

## Description

Swift-protobuf build failed with command suggested in their Readme:
```swift build -c release -Xswiftc -static-stdlib```
but it works with 
```swift build --static-swift-stdlib -c release```

## Testing instructions

Run ./tools/install-dependencies.

I'm not sure if it will work fine with previous Swift version.
But CI job should answer this question.

## Types of changes

* Bug fix (non-breaking change which fixes an issue)

## Checklist
